### PR TITLE
Fix RPCServer configuration falure

### PR DESCRIPTION
**Pull request by Qiao-Jin**
Close [#1531](https://github.com/neo-project/neo/issues/1531)

### CODE REVIEW

File: src/RpcServer/RpcServer.cs [hunk 1]
```diff
@@ -55,7 +57,22 @@ private bool CheckAuth(HttpContext context)
...
+                try
+                {
+                    config = GetConfiguration();
+                }
+                catch (FormatException)
+                {
+                    remainingTimes--;
+                    Thread.Sleep(10);
+                }
+            }
```
Comment (erikzhang) [inline src/RpcServer/RpcServer.cs:73]: I think we can just catch the exception and ignore it.

Comment (Qiao-Jin) [reply src/RpcServer/RpcServer.cs:73]: If we ignore the exception without try once more, a null config will be loaded, which will give rise to a null-pointer exception and crash the node

Comment (erikzhang) [reply src/RpcServer/RpcServer.cs:73]: If you don't invoke `Settings.Load()`, then it won't.

Comment (Qiao-Jin) [reply src/RpcServer/RpcServer.cs]: Got it, have changed correspondingly

**Comment by erikzhang** [comment]
Please test it. If it works, we can update all the projects.

**Comment by shargon** [comment]
Without any log it's hard to know when something was wrong, you can think that it's well configured and it's not

**Comment by Qiao-Jin** [comment]
> Without any log it's hard to know when something was wrong, you can think that it's well configured and it's not

I'm not sure whether we need to log it, @erikzhang could you give us some advice?
https://github.com/neo-project/neo/issues/1531#issuecomment-610233266

**Comment by erikzhang** [comment]
https://github.com/neo-project/neo/blob/bf16df9d5a090dfcaff90d7c9d5454cf5ad09101/src/neo/Plugins/Plugin.cs#L69

How about we catch exceptions here? @Qiao-Jin @shargon

**Comment by Qiao-Jin** [comment]
> https://github.com/neo-project/neo/blob/bf16df9d5a090dfcaff90d7c9d5454cf5ad09101/src/neo/Plugins/Plugin.cs#L69
> 
> How about we catch exceptions here? @Qiao-Jin @shargon

I think it's good to catch exceptions here

**Comment by shargon** [comment]
The code looks well but I prefer to know get the error, otherwise it's impossible to know where it fail.

**Pull request closed without merging**